sklearn/manifold/tests/test_isomap.py

```diff
@@ -204,6 +230,35 @@ def test_sparse_input():
             clf.fit(X)
 
 
+def test_isomap_fit_precomputed_radius_graph():
+    X, y = datasets.make_s_curve(200, random_state=0)
+    radius = 10
+
+    g = neighbors.radius_neighbors_graph(X, radius=radius, mode="distance")
+    isomap = manifold.Isomap(n_neighbors=None, radius=radius, metric="precomputed")
+    isomap.fit(g)
+    precomputed_result = isomap.embedding_
+
+    isomap = manifold.Isomap(n_neighbors=None, radius=radius, metric="minkowski")
+    result = isomap.fit_transform(X)
+    # test fit_transform yields similar result as using precomputed distance matrix
+    assert np.allclose(precomputed_result, result)
+
+
+def test_isomap_raise_error_when_neighbor_and_radius_both_set():
+    X, _ = datasets.load_digits(return_X_y=True)
+
+    # works fine when one of `n_neighbors` and `radius` is provided
+    for n_neighbors, radius in [(None, 2), (2, None)]:
+        isomap = manifold.Isomap(n_neighbors=n_neighbors, radius=radius)
```

Comment:
We do not need to include this here since other tests already checks for these parameters.